frontend/tests/integration/components/transportation/tdf/modal-splits-test.js

```diff
@@ -86,4 +86,222 @@ module('Integration | Component | transportation/tdf/modal-splits', function(hoo
     assert.equal(this.element.querySelector('[data-test-total-pm]').textContent, '11', 'total pm calculated');
     assert.equal(this.element.querySelector('[data-test-total-saturday]').textContent, '15', 'total saturday calculated');
   });
+
+  test('allPeriod total calculates correctly when user manually inputs mode split values', async function(assert) {
+    const store = this.owner.lookup('service:store');
+
+    this.server.create('user');
+    this.server.create('data-package');
+    this.server.create('project', {
+      publicSchoolsAnalysis: this.server.create('publicSchoolsAnalysis'),
+      transportationAnalysis: this.server.create('transportationAnalysis', {
+        transportationPlanningFactors: [
+          this.server.create('transportationPlanningFactor', {
+            dataPackage: this.server.create('dataPackage', 'nycAcs'),
+            landUse: 'residential',
+            // necessary for displaying certain elements
+            // Temporal Splits tab (true) vs. All Periods tab (false)
+            temporalModeSplits: false,
+            // necessary for displaying certain elements
+            // user input mode splits (true) vs. mode split values from census tract calculator (false)
+            manualModeSplits: true,
+          }),
+        ],
+      }),
+    });
+
+    // define project model
+    const project = await store.findRecord('project', 1, {
+      include: ['transportation-analysis,transportation-analysis.transportation-planning-factors'].join(','),
```

Comment:
That .join isn't necessary because there's only 1 element in this array

Reply:
There's two elements, `transportation-analysis` and `transportation-analysis.transportation-planning-factors`

Reply:
I'm finding that `['transportation-analysis,transportation-analysis.transportation-planning-factors'].length = 1 `

There isn't a `'` separating that comma.

Reply:
Just tested it out, without he `.join()` tests will fail, so it is necessary

Reply:
It doesn't need to be in an array :)

Reply:
Oohh okay I see what you're saying